src/core/backbone_router/multicast_listeners_table.cpp

```diff
@@ -308,6 +308,18 @@ Error MulticastListenersTable::GetNext(Listener::Iterator &aIterator, Listener::
     return error;
 }
 
+bool MulticastListenersTable::HasListener(const Ip6::Address &aAddress) const
+{
+    for (uint16_t i = 0; i < mNumValidListeners; i++)
+    {
+        if (mListeners[i].GetAddress() == aAddress)
+        {
+            return true;
+        }
+    }
+    return false;
```

Comment:
Change to use single `return`? (part of our style guide).

Reply:
Done